@kiva/kv-loan-filters/src/loanAmount.ts

```diff
@@ -0,0 +1,59 @@
+import { getMinMaxRangeFilter, createMinMaxRange } from './minMaxRangeUtils';
+
+/**
+ * The min risk rating value
```

Comment:
copy pasta

Reply:
good catch. I'm also open to suggestions on the min max loanAmount here. I figured anything we pass through this code a present would fit within these bounds.

Reply:
I think this is a good range. We could still theoretically go down to 5, but 25 is our regular note size so I think that is fine. 
I think our max LSE loan has been 500,000, and there have only been 7 of these. I think I've heard that that amount is a little on the high end for lenders so I don't see us going too far beyond this. We can set the max here, or a little higher. 1,000,000 will definitely cover it.